ksql-execution/src/main/java/io/confluent/ksql/execution/codegen/SqlToJavaVisitor.java

```diff
@@ -712,6 +722,25 @@ private String buildSupplierCode(String typeString, String code) {
       }
     }
 
+    @Override
+    public Pair<String, SqlType> visitStructExpression(StructExpression node, Void context) {
+      final String schemaName = structToCodeName.apply(node);
+      final StringBuilder struct = new StringBuilder("new Struct(").append(schemaName).append(")");
+      for (Entry<String, Expression> field : node.getStruct().entrySet()) {
+        struct.append(".put(")
+            .append('"')
+            .append(field.getKey())
+            .append('"')
+            .append(",")
+            .append(process(field.getValue(), context).getLeft())
+            .append(")");
+      }
+      return new Pair<>(
+          "((Struct)" + struct.toString() + ")",
```

Comment:
think the cast is superfluous: 
```suggestion
          "(" + struct.toString() + ")",
```

Reply:
I'm following the pattern in the rest of the code base here so I'll leave it as is